src/Standards/Generic/Docs/CodeAnalysis/JumbledIncrementerStandard.xml

```diff
@@ -1,7 +1,7 @@
-<documentation title="Jumbled Incrementers">
+<documentation title="Jumbled Increments">
```

Comment:
I don't think this one should be changed as it refers to the name of the sniff (and renaming the sniff would break BC).

Reply:
Fixed - I reverted other fixes that wouldn't work before submitting this but missed that